README.md

```diff
@@ -92,7 +92,8 @@ This project makes use of multiple other projects. We wouldn't be here without t
 * [Ocarina of Time Randomizer](https://github.com/TestRunnerSRL/OoT-Randomizer)
 
 ## Contributing
-For contribution guidelines, please see our [Contributing doc.](/docs/contributing.md)
+To contribute to Archipelago, including the WebHost, core program, or by adding a new game, see our
+[Contributing guidelines.](/docs/contributing.md)
```

Comment:
In general, it is a good idea not to include the punctuation in a hyperlink. I think also changing the last two links to match this would be reasonable, even if perhaps out-of-scope.
```suggestion
[Contributing guidelines](/docs/contributing.md).
```